src/main/webapp/WEB-INF/views/reservation-lookup.jsp

```diff
@@ -1,14 +1,165 @@
 <!-- Purple Team: D. Bonis, R. Duvall, M. Guthman, O.Tsolmon -->
 
+<!-- Author: M.Guthman
+Date: 09/13/2023 -->
+
 <!DOCTYPE html>
-<html lang="en">
+<html>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+
+	<head>
+		<title>Reservation Lookup</title>
+		<link rel="stylesheet" type="text/css" href="css/reservationLookupStyle.css">
+		<link rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+		<link rel="stylesheet" href="https://use.typekit.net/lot0wjb.css">
+	</head>
+
+	<body>
+		<header>
+			<a href="home"><img id=logo src="_images/MB_Title_Logo_3.png"></img></a>
+			<!-- USER ICON-->
+			<a href="login"><i class="fa fa-user-circle-o"
+					style="float: right; font-size: 56px; color: #554D41"></i></a>
+			<div class="navbar">
+
+				<div class="nav-grid-container">
+
+					<div class="nav-grid-item">
+
+					</div>
+
+					<div class="nav-grid-item">
+
+						<a href="about-us" target="_self">ABOUT US</a>
+					</div>
+
+					<div class="nav-grid-item">
+
+						<a href="attractions" target="_self">ATTRACTIONS</a>
+
+					</div>
+
+					<div class="nav-grid-item">
+
+						<a href="look-up-reservation" target="_self">FIND RESERVATION</a>
```

Comment:
Currently, the link to the find reservation page is broken. I think we need to change the href to "lookup" instead of look-up-reservation.